.env.test

```
@@ -11,3 +11,6 @@ SMTP_HOST=smtp.gmail.com
 SMTP_PORT=587
 SMTP_USER=user@test.com.ar
 SMTP_PASS=password
+
+LOGIN_EXPIRATION_DAYS=30
```

Esto puede joder con el experimento, lo pondría por defecto en algo más laxo:

```suggestion
LOGIN_EXPIRATION_DAYS=360
```

---

Por cierto, para no olvidar setear estos envs, ya pasé por containerization a actualizar el valor: https://github.com/fundacion-sadosky/containerization/blob/master/pilasbloques/api/.env.example

Y ya que estaba lo modifiqué en los .env de staging y de prod. (De nada, gente del futuro, de nada)